cli-output/src/display.rs

```diff
@@ -791,6 +791,7 @@ mod test {
                 program_id: Pubkey::new_from_array([2u8; 32]),
                 data: vec![1, 2, 3],
             }),
+            compute_units_consumed: None,
```

Comment:
Do you mind displaying the consumed compute units in `write_transaction` and then add a value here to test it?

Reply:
@jstarry Do you have any preference where CUs should be mentioned? Order of fields is important for UX, imo. 
I'd put it just above `Log Messages`

For reference

```
Block Time: 2021-08-10T22:16:31Z
Version: legacy
Recent Blockhash: 11111111111111111111111111111111
Signature 0: 5pkjrE4VBa3Bu9CMKXgh1U345cT1gGo8QBVRTzHAo6gHeiPae5BTbShP15g6NgqRMNqu8Qrhph1ATmrfC1Ley3rx (pass)
Account 0: srw- 4zvwRjXUKGfvwnParsHAS3HuSVzV5cA4McphgmoCtajS (fee payer)
Account 1: -r-x 4vJ9JU1bJJE96FWSJKvHsmmFADCg4gpZQff4P3bkLKi
Instruction 0
  Program:   4vJ9JU1bJJE96FWSJKvHsmmFADCg4gpZQff4P3bkLKi (1)
  Account 0: 4zvwRjXUKGfvwnParsHAS3HuSVzV5cA4McphgmoCtajS (0)
  Data: []
Status: Ok
  Fee: ◎0.000005
  Account 0 balance: ◎0.000005 -> ◎0
  Account 1 balance: ◎0.00001 -> ◎0.0000099
Log Messages:
  Test message
Return Data from Program 8qbHbw2BbbTHBW1sbeqakYXVKRQM8Ne7pLK7m6CVfeR:
  Length: 3 (0x3) bytes
0000:   01 02 03                                             ...
Rewards:
  Address                                            Type        Amount            New Balance         \0
  4vJ9JU1bJJE96FWSJKvHsmmFADCg4gpZQff4P3bkLKi        rent        -◎0.000000100     ◎0.000009900       \0
```

Reply:
Yeah, that looks like a good spot for it